testhelper/testhelper.go

```diff
@@ -38,6 +47,8 @@ func SetupDeps(sensors []string) resource.Dependencies {
 
 	for _, sensor := range sensors {
 		switch sensor {
+		case "stub_lidar":
```

Comment:
Why add "stub_lidar" instead of either using "invalid_sensor" or renaming "invalid_sensor"?

Reply:
Because we don't plan on having this lidar ever be called during the test so semantically, it is not an "invalid lidar".

Reply:
@kim-mishra I've updated it per your feedback. Good question!